docs/docs/dev-manuals/services/discount.mdx

```diff
@@ -63,41 +63,41 @@ The following types are used in the defined events:
 
 ```tsx
 interface Order {
-  id: string,
-  userId: string,
-  createdAt: Date,
-  orderStatus: OrderStatus,
-  placedAt?: Date,
-  rejectionReason?: RejectionReason,
-  orderItems: OrderItem[],
-  shipmentAddressId: string,
-  invoiceAddressId: string,
-  compensatableOrderAmount: number,
-  paymentInformationId: string
+ id: string;
```

Comment:
so I know, formatter and so on, but indentation with one space is imho not enough

Reply:
also I configured prettier to not format embedded languages in the package.json: `--embedded-language-formatting=off`
is it possible that your format plugin ignores the settings configured by the repository? maybe you want to check this in your settings

Reply:
The settings were set to `tabWidth: 1`, fixed it with the new commit [3c0c094](https://github.com/MiSArch/docs/pull/122/commits/3c0c094e1118b10dd31ed1b70bee46431493897a)